app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -235,30 +233,29 @@ abstract class BaseBrowserFragment :
             )
         }
 
-        view
+        return view
     }
 
-    final override fun onViewCreated(view: View, savedInstanceState: Bundle?) =
-        PerfStartup.baseBfragmentOnViewCreated.measureNoInline { // weird indentation to avoid breaking blame.
-            initializeUI(view)
+    final override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        // weird indentation to avoid breaking blame.
```

Comment:
nitpick (non-blocking): in your next PR, could you remove this comment? It was actually written for the `measureNoInline` call (iirc, originally the method body was not indented and the line had a weird indentation to cause the linter to be happy but the method body has since been autoformatted such that the comment was irrelevant).

Even though it's more related to this PR than the next one, I don't want to add it to this PR because it'd create add an additional delay that adds overhead for both of us. There's a certain standard of correctness we want to maintain for our project but we need to make deliberate trade-offs about the amount of time required to maintain them. In this case, which commit/PR a comment removal ends up probably isn't going to cause bugs or confusion down the line so it's fine to remove it wherever.

Reply:
Sure will remove this comment on next PR